woocommerce/wc-admin
# Allow renaming Dashboard sections

Fixes #2120.

### Accessibility


### Screenshots
![dashboard-titles](https://user-images.githubusercontent.com/3616980/56916190-9b16cc00-6ab8-11e9-9ebc-a20df4e9c18d.gif)

### Detailed test instructions:
- Go to the _Dashboard_.
- Open any section ellipsis menu and verify there is an input text to rename the section.
- Rename the section and verify the title is updated when you unfocus the input.
- Rename the section again leaving the input empty.
- Verify instead of an empty title, you get the default title.
- Verify when you close/open the ellipsis menu, the input text was also reset to the default title.
